mlblocks_primitives/mlprimitives.timeseries.aggregate_average_time.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "mlprimitives.timeseries.aggregate_average_time",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "description": "mlprimitives.timeseries.aggregate_average_time",
+    "classifiers": {
+        "type": "preprocessor",
+        "subtype": "feature_extractor"
+    },
+    "modalities": ["timeseries"],
+    "primitive": "mlprimitives.timeseries.aggregate_average_time",
+    "produce": {
+        "args": [
+            {
+                "name": "df_time_value",
+                "type": "Pandas.DataFrame"
```

Comment:
call this "X" and use the "keyword" entry to make it match the actual argument name.
Also use lowercase for `pandas`.
So:
```
{
    "name": "X",
    "keyword": "time_value",
    "type": "pandas.DataFrame"
}
```